terraform/environments/mlra/modules/cloudwatch-alarms/main.tf

```diff
@@ -0,0 +1,197 @@
+#resource "aws_cloudwatch_metric_alarm" "cwalarms" {
+#  alarm_name                = var.alarm_name.value
+#  comparison_operator       = var.oper.value
+#  evaluation_periods        = var.eval.value
+#  metric_name               = var.metric_name.value
+#  namespace                 = var.namespace.value
+#  period                    = var.period.value
+#  statistic                 = var.stat.value
+#  threshold                 = var.thresh.value
+#  alarm_description         = var.alarmdesc.value
+#  insufficient_data_actions = []
+#}
+
+
+resource "aws_cloudwatch_metric_alarm" "esccpuoverthreshold" {
+  alarm_name         = "ECS-CPU-high-threshold-alarm"
+  alarm_description  = "If the CPU exceeds the predefined threshold, this alarm will trigger. \n Please investigate."
+  namespace          = "AWS/ECS"
+  metric_name        = "CPUUtilization"
+  statistic          = " Average "
+  period             = "60"
+  evaluation_periods = "5"
+  threshold          = var.pECSCPUAlarmthreshold.value
+  treat_missing_data = "breaching"
+  Dimensions = {
+    ClusterName = var.pClusterName.value
+  }
+  comparison_operator = "GreaterThanthreshold"
+}
+
+resource "aws_cloudwatch_metric_alarm" "ecsmemoryoverthreshold" {
+  alarm_description  = " If the memory util exceeds the predefined threshold, this alarm will trigger.\n Please investigate."
+  namespace          = " AWS/ECS"
+  metric_name        = " MemoryUtilization"
+  statistic          = " Average"
+  period             = " 60"
+  evaluation_periods = " 5"
+  threshold          = var.pECSMemoryAlarmthreshold.value
+  treat_missing_data = " breaching"
+  Dimensions = {
+    ClusterName = var.pClusterName.value
```

Comment:
Just a flag, as I realise this can't be tested until the ECS work is more or less complete - the .value may not be required, albeit it may be a different attribute altogether. Applies to all instances where .value attribute appears.